### DISCUSSION TITLE
Consider letting the user supply their own glamour theme

### DISCUSSION THREAD

**Discussion by h0adp0re** [ankitpokhrel/jira-cli#316] [Ideas] [answered]
Since you're using glamour for rendering markdown, it'd be real neat if we could use a theme we're already used to.

**Comment by realtime-neil** [comment]
...even a theme that has no colors at all. Hint, hint: https://github.com/ankitpokhrel/jira-cli/issues/291

**Comment by ankitpokhrel** [comment, marked as answer]
@h0adp0re @realtime-neil with the change in #317, `jira issue view` should respect `GLAMOUR_STYLE` env. `--plain` mode will always use `notty`.

You can test the change by reinstalling the tool with following command.
```sh
go install github.com/ankitpokhrel/jira-cli/cmd/jira@request-316
```

**Reply by realtime-neil** [reply]
@ankitpokhrel looks good

**Reply by realtime-neil** [reply]
@ankitpokhrel does this fix #286 also?

**Reply by ankitpokhrel** [reply]
@realtime-neil thanks for checking. Regarding #286 looks like it is fixed with library upgrades I did recently 🙈

**Reply by h0adp0re** [reply]
thanks! quick as always 💪
